### DISCUSSION TITLE
keep fork up to date with renovate

### DISCUSSION THREAD

**Discussion by childnode** [renovatebot/renovate#20713] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

While `includeForks[=false]` option only controls, which repositories are scanned for updates, I'm asking myself if I can use renovate to update a forked repository.

Because `includeForks=false` is a good default to mitigate fork get's pull requests that are already / in parallel maintained on source and should get into the fork by MR itself.

So said, this is a nasty manual task and noone likes it neither do it right ;)

Back to the question what I expect to get from renovate:

if renovate detects a forked repository while includeForks is set false it should check if it can fast-forward forked branches, if not create a MR from the origin to the forked branch with the update.

### What you tried so far.

doing it manually .. 🤮

since [gitlab pull mirror feature is a premium feature only](https://docs.gitlab.com/ee/user/project/repository/mirror/#:~:text=Pull%3A%20Mirror%20a%20repository%20from%20another%20location%20to%20a%20GitLab%20Premium%20instance.)

**Comment by viceice** [comment, marked as answer]
not possible

**Reply by childnode** [reply]
I had fear to hear that ;/ thx
